modules/snippets/deprecated-feature.adoc

```diff
@@ -0,0 +1,14 @@
+// When including this file, ensure that {FeatureName} is set immediately before
```

Comment:
All of these snippets are under modules/snippets instead of just /snippets. I think they need moved to the proper location and then the modules/snippets directory should be removed